packages/apps/reputation-oracle/server/src/modules/qualification/qualification.controller.ts

```diff
@@ -0,0 +1,103 @@
+import {
+  Controller,
+  Post,
+  Get,
+  Delete,
+  Body,
+  Param,
+  UseGuards,
+  HttpCode,
+} from '@nestjs/common';
+import {
+  CreateQualificationDto,
+  AssignQualificationDto,
+  UnassignQualificationDto,
+} from './qualification.dto';
+
+import {
+  ApiBearerAuth,
+  ApiTags,
+  ApiOperation,
+  ApiResponse,
+  ApiBody,
+} from '@nestjs/swagger';
+import { JwtAuthGuard, RolesAuthGuard } from '../../common/guards';
+import { QualificationService } from './qualification.service';
+import { Roles } from 'src/common/decorators';
+import { Role } from 'src/common/enums/user';
+
+@ApiTags('Qualification')
+@Controller('qualification')
+@ApiBearerAuth()
+@UseGuards(JwtAuthGuard, RolesAuthGuard)
+export class QualificationController {
+  constructor(private readonly qualificationService: QualificationService) {}
+
+  @Post()
+  @Roles(Role.ADMIN)
+  @HttpCode(201)
+  @ApiOperation({ summary: 'Create a new qualification' })
+  @ApiBody({ type: CreateQualificationDto })
+  @ApiResponse({
+    status: 201,
+    description: 'Qualification created successfully',
+  })
+  @ApiResponse({ status: 400, description: 'Bad Request' })
+  create(@Body() createQualificationDto: CreateQualificationDto) {
+    return this.qualificationService.createQualification(
+      createQualificationDto,
+    );
+  }
+
+  @Get()
+  @Roles(Role.OPERATOR, Role.WORKER, Role.HUMAN_APP, Role.ADMIN)
```

Comment:
I think it should use a signature auth guard instead of JWT.

Reply:
https://human-protocol.gitbook.io/hub/human-tech-docs/architecture/protocol-features/qualifications-system#get-list-of-qualifications

Reply:
cc @portuu3